File: TownOfSalemCalculator/src/townofsalemcalculator/Conditions/Concrete/KillMessages/WerewolfKill.java
package townofsalemcalculator.Conditions.Concrete.KillMessages;

import java.util.Arrays;
import java.util.List;
import static townofsalemcalculator.Role.Werewolf;

/**
 * The Concrete Condition that a Werewolf has killed someone
 * @author Multifacio
 * @version 1.1
 * @since 2017-2-10
 */
public class WerewolfKill extends KillMessage {
    public WerewolfKill() {
        super(Werewolf);
    }
    
    @Override
    public String getCondition() {
        return "He was mauled by a Werewolf";
    }

    @Override
    public List<String> keyWords() {
        List<String> keyWords = Arrays.asList(new String[]{"He", "Died", "Killed", "Death", "Mauled"});
        keyWords.addAll(Werewolf.getKeyWords());
        return keyWords;
    }
}

File: src/Conditions/Concrete/KillTypes/WerewolfKill.py
from src.Concepts.Role import Role
from src.Conditions.Concrete.ExistenceCondition import ExistenceCondition
from typing import FrozenSet

class WerewolfKill(ExistenceCondition):
    """ The Werewolf Kill condition is used if some player was killed by a Werewolf. """

    def __init__(self, amnesiacRemembered: FrozenSet[Role] = None):
        super().__init__(frozenset({Role.WEREWOLF}), amnesiacRemembered)